<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNManageEthenetWidget</class>
 <widget class="QWidget" name="HNManageEthenetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetIP" native="true">
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select network</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>IP configure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_curnet">
        <property name="text">
         <string>Current:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="7">
       <widget class="HNWIFIWidget" name="tableView_Wifi"/>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="chk_dhcp">
        <property name="text">
         <string>Use DHCP</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ip address:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEdit_ip"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mask:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEdit_mask"/>
      </item>
      <item row="4" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gateway:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEdit_gateway"/>
      </item>
      <item row="6" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DNS:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="lineEdit_dns"/>
      </item>
      <item row="7" column="3">
       <widget class="HNPushButton" name="btn_saveip">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>119</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNPushButton</class>
   <extends>QPushButton</extends>
   <header>hnpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>HNWIFIWidget</class>
   <extends>QTableView</extends>
   <header>hnwifiwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
